erc/ERCS/erc-7562.md

```diff
@@ -129,10 +221,10 @@ To help make sense of these params, note that a malicious paymaster can at most
 ### Running the Validation Rules
 
 1. A block builder or a bundler should perform a full validation before accepting a `UserOperation` into its mempool.
-2. During validation phase, the bundler should trace the execution and apply all the rules defined in this document.
+2. During the validation phase, the bundler should trace the execution and apply all the rules defined in this document.
 3. A bundler should also perform a full validation of the entire bundle before submission.
 4. The validation rules prevent an unstaked entity from detecting the bundle validation.
-   However, it is possible for a malicious staked entity to detect the bundle validation and cause a revert.
+   However, a malicious staked entity can detect it runs in a bundle validation and cause a revert.
 5. The failed `UserOperation` should be dropped from the bundle.
 6. The staked entity that caused a revert violated the Account Abstraction rules and should be marked as `THROTTLED`.
```

Comment:
I think that we should simplify these rules: 
We have a definition of status: OK/THROTTLED/BANNED. 
For banned/throttled, we have a requirement of opsSeen vs opsIncluded, with a certain slack. 
UREP-020 is meant only for OK entities opsAllowed calculation. 
Once the entity is throttled, i.e. `max_seen > opsIncluded + THROTTLING_SLACK`, we no longer calculate opsAllowed by UREP-020, but allowing only `THROTTLED_ENTITY_MEMPOOL_COUNT`  and `THROTTLED_ENTITY_BUNDLE_COUNT` by SREP-030. If it keeps misbehaving, it may get banned, which then SREP-020 handles. 
This should hold for both staked and unstaked entities imo.

Reply:
I'm confused. I don't think we have status defined anywhere. It is just implied by the opsSeen/opsIncluded ratio. So the only way to force a certain status is to manipulate these two numbers.

I thought UREP-020 is only for unstaked entities, which means that these entities are always "OK" unless UREP changes their numbers.  Do you mean that we keep SREP for unstaked entities as well, and use it in addition to UREP?  In that case SREP is no longer the right name for it, but I didn't think staked reputation applies to unstaked entities.